examples/bluesky-firehose/watch.py

```
@@ -0,0 +1,101 @@
+    print(f"-> {sql}\n", flush=True)
+
+    n = 0
+    t0 = last = time.time()
+    try:
+        with psycopg.connect(DSN, autocommit=True) as conn, conn.cursor() as cur:
+            stream = cur.stream(sql)
+            names = None
+            for row in stream:
+                if names is None:
+                    names = [d.name for d in cur.description]
+                cols = dict(zip(names, row))
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)